# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295051047 28800
#      Fri Jan 14 16:24:07 2011 -0800
# Node ID a4b0e292107f3884de0136dec200cde7311027f9
# Parent  702dbd0c53f5bc8ffb671213cd9ee81e5822cc83
Add undocumented function nfields to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-14  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/container.txi: Add undocumented nfields to documentation.
+
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/poly.txi: Add undocumented ppder, ppint, ppjumps
 	functions to documentation.
 
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add undocumented matlabroot to documentation.
diff --git a/doc/interpreter/container.txi b/doc/interpreter/container.txi
--- a/doc/interpreter/container.txi
+++ b/doc/interpreter/container.txi
@@ -438,28 +438,30 @@ structure or a structure array.
 
 @DOCSTRING(isstruct)
 
 @node Manipulating Structures
 @subsection Manipulating Structures
 
 Other functions that can manipulate the fields of a structure are given below.
 
-@DOCSTRING(rmfield)
-
-@DOCSTRING(setfield)
-
-@DOCSTRING(orderfields)
+@DOCSTRING(nfields)
 
 @DOCSTRING(fieldnames)
 
 @DOCSTRING(isfield)
 
 @DOCSTRING(getfield)
 
+@DOCSTRING(setfield)
+
+@DOCSTRING(rmfield)
+
+@DOCSTRING(orderfields)
+
 @DOCSTRING(substruct)
 
 @node Processing Data in Structures
 @subsection Processing Data in Structures
 
 The simplest way to process data in a structure is within a @code{for}
 loop (@pxref{Looping Over Structure Elements}).  A similar effect can be
 achieved with the @code{structfun} function, where a user defined
